### PR TITLE
UserService_EditUser added

### PR DESCRIPTION
**Pull request by VLiamin**

### CODE REVIEW

**Comment by nmamro** [comment]
Check request name and name convention

File: Services/UserService/Commands/EditUserCommand.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+﻿using DTO;
+using Kernel;
+using System;
+using UserService.Interfaces;
+
+namespace UserService.Commands
+{
+    public class EditUserCommand : IEditUserCommand
```
Comment (Cerberous007) [inline Services/UserService/Commands/EditUserCommand.cs:8]: I think, you should inject this command in startup file

File: Services/UserService/Commands/EditUserCommand.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    {
+        public void Execute(UserEmailPassword userEmailPassword)
+        {
+            try
+            {
+                CommonValidations.ValidateEmail(userEmailPassword.Email);
+                editUser(userEmailPassword);
```
Comment (Cerberous007) [inline Services/UserService/Commands/EditUserCommand.cs:16]: This command edit only UserEmailPassword data. Ask frontend team  what data needs to be changed

**Review by Cerberous007 — CHANGES_REQUESTED** [review]

File: Services/UserService/Commands/EditUserCommand.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+using Kernel;
+using System;
+using UserService.Interfaces;
+
+namespace UserService.Commands
+{
+    public class EditUserCommand : IEditUserCommand
+
+    {
+        public void Execute(UserEmailPassword userEmailPassword)
```
Comment (strannick-rodion) [inline Services/UserService/Commands/EditUserCommand.cs:11]: Why logic only for mode UserEmailPassword?

File: Services/UserService/Commands/EditUserCommand.cs [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        {
+            try
+            {
+                CommonValidations.ValidateEmail(userEmailPassword.Email);
+                editUser(userEmailPassword);
+                return;
```
Comment (strannick-rodion) [inline Services/UserService/Commands/EditUserCommand.cs:17]: You should use TODO for something not ending

**Review by strannick-rodion — CHANGES_REQUESTED** [review]
Change name of pull request

**Comment by nmamro** [comment]
We need to be able edit next data: Name, Surname, Date of birth, Password, not UserEmailPassword stuff. What is more, we cannot change email at all.

**Pull request closed without merging**